sql-plan-management.md

```diff
@@ -350,11 +427,11 @@ create global binding for select * from t where a < 100 and b < 100 using select
 
     ```sql
     -- 在待升级的版本上检查自动演进的开关 `tidb_evolve_plan_baselines` 是否关闭。
-  
+
```

Comment:
The editor automatically formats this part of the content. Although it is a useless change, can it be adopted if it conforms to the specification?

Reply:
LGTM